ruma-common/src/push/predefined.rs

```diff
@@ -146,6 +146,18 @@ impl OverridePushRule {
             ],
         })
     }
+
+    /// MSC2677: Annotations and Reactions
```

Comment:
It's good to mention the MSC, but the docs should also explain what this rule does (like the docs for the other rules).